app/Note.php

```diff
@@ -4,35 +4,60 @@
 
 use App\Helpers\DateHelper;
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Parsedown;
```

Comment:
This ships with Laravel's mail component, but I also added it to composer.json now.